prebid-mobile/modules/rendering/android-sdk-integration-pb.md

```diff
@@ -37,9 +37,16 @@ In this mode, the developer is responsible for the following actions:
 This approach is available for the following ad formats:
 
 - Display Banner via `BannerAdUnit`
-- Video Banner and Instream Video via `VideoAdUnit`
+- Video Banner and Instream Video via `InStreamVideoAdUnit`
```

Comment:
Android and iOS docs differ; is it expected? Check out  [ios-sdk-integration-pb.md](https://github.com/prebid/prebid.github.io/pull/5982/files#diff-fbe320d690eb145536e1521034dcb5b270246edfeba610cc653863c3203d288fR40)